### PR TITLE
:robot: Package update

### PR DESCRIPTION
**Pull request by vm-packages**
Automated package update

### CODE REVIEW

**Comment by emtuls** [comment]
I think we've hit this error with `ImHex` before due to using Linux host vs Windows host if I recall, so we may either need to test it on a Windows Host to verify it works, or remove the package. I'm okay with keeping it (if it works), just noting this issue.
image

For `System-Informer`, there is a weird issue. It detected that it should upgrade to `3.1.7741`, but that does not exist. The only update that has occurred for that tool would be from `3.0.7645` to `3.0.7660`. Not really sure what happened that @Ana06 (for awareness)

**Comment by Ana06** [comment]
The issue with ImHex is known: https://github.com/mandiant/VM-Packages/issues/940 We should consider removing that package to fix #940, but we should do it outside of this PR as it is completely unrelated.

**Comment by Ana06** [comment]
I am also not sure why `System-Informer`, I have re-triggered the package update action to see if it happens again (just in case that URL did work at the time of the update).


Documented packages @emtuls tested just in case they change with the new update:
image

**Comment by Ana06** [comment]
@emtuls System-Informer is now not included, so it seems like the URL did work at some point.

We new to test the following tools as they were not included in the first update you tested:

- [ ] [Update capa.vm to 7.1.0](https://github.com/mandiant/VM-Packages/pull/1076/commits/86c368b2f1e4ccad7af17b45d9e0189ed9fb4ac2)
- [ ] [Update cmder.vm to 1.3.25](https://github.com/mandiant/VM-Packages/pull/1076/commits/c621d2a05f06593e65191538ddc14cd8fdebbefa)
- [ ] [Update exiftool.vm to 12.87.0](https://github.com/mandiant/VM-Packages/pull/1076/commits/294e0c64d4c6ce44dc93e8e3ed63e0ff8e150dfc)
- [ ] [Update ghidra.vm to 11.1.1](https://github.com/mandiant/VM-Packages/pull/1076/commits/3d0ec492d635a4d5b783c43c319d2ebddf9acfbb)
- [ ] [Update ida.plugin.hashdb.vm to 1.9.1](https://github.com/mandiant/VM-Packages/pull/1076/commits/fd0b8a9526ef638776830222babeb29038570a0c)
- [ ] [Update vscode.vm to 1.90.1](https://github.com/mandiant/VM-Packages/pull/1076/commits/8a7772ca0bf090fb394c52068dd28978c7c81506)

@emtuls do you have time to test it today or tomorrow (to avoid the new update on Wednesday makes this bigger and more tricky to test)? :smile:

**Comment by emtuls** [comment]
@Ana06 All of the packages work except for `ida.plugin.hashdb.vm`.

`ida.plugin.hashdb.vm` requires `python3.vm` to be installed as it uses `VM-Pip-Install`, so we need to adjust the dependencies in the `.nuspec`. :)

File: packages/ida.plugin.hashdb.vm/ida.plugin.hashdb.vm.nuspec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>ida.plugin.hashdb.vm</id>
-    <version>1.9.0</version>
+    <version>1.9.1</version>
     <authors>OALabs</authors>
     <description>Malware string hash lookup plugin for IDA Pro</description>
     <dependencies>
```
Comment (emtuls) [inline packages/ida.plugin.hashdb.vm/ida.plugin.hashdb.vm.nuspec:8]: We need to add `<dependency id="python3.vm" />` due to the `pip install requests` requirement in the installer.

**Review by emtuls — CHANGES_REQUESTED** [review]

**Review by emtuls — APPROVED** [review]

**Comment by emtuls** [comment]
I'm going to push this through and open a PR to fix `ida.plugin.hashdb.vm`

**Pull request merged**